jpos/src/main/java/org/jpos/rc/CMF.java

```diff
@@ -67,7 +67,7 @@ public enum CMF implements IRC {
     INVALID_PINBLOCK(1026),
     PIN_LENGTH_ERROR(1027),
     PIN_KEY_SYNC_ERROR(1028),
-    SUSPECTED_COUNTERFEIT_CARD(1029),
+    SUSPECTED_COUNTERFEIT_CARD_NOT_PICKUP(1029),
```

Comment:
```suggestion
    SUSPECTED_COUNTERFEIT_CARD_DONT_PICKUP(1029),
```

The intention of the message is that there is a suspected fraud, but we shouldn't seize the card.

Reply:
@aVolpe your suggestion has been applied